### PR TITLE
[Theia] Upgrade version of less to 3.0.3 for fixing the vulnerability in cryptiles package

### PR DESCRIPTION
**Pull request by vinayb21**
Fixes: https://github.com/theia-ide/theia/issues/4440

### CODE REVIEW

File: dev-packages/application-manager/package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "font-awesome-webpack": "0.0.5-beta.2",
     "fs-extra": "^4.0.2",
     "ignore-loader": "^0.1.2",
-    "less": "^2.7.2",
+    "less": "^3.0.3",
```
Comment (akosyakov) [inline dev-packages/application-manager/package.json]: @marcdumais-work CQ?

Comment (akosyakov) [inline dev-packages/application-manager/package.json]: @marcdumais-work CQ?

**Comment by akosyakov** [comment]
@vinayb21 please commit changes to yarn.lock file

File: dev-packages/application-manager/package.json (hunk 1 above)
Comment (vinayb21) [reply dev-packages/application-manager/package.json]: @akosyakov done.

**Comment by kittaakos** [comment]
@vinayb21, FYI the build has failed because of this https://github.com/theia-ide/theia/issues/3787.

**Comment by vinayb21** [comment]
> @vinayb21, FYI the build has failed because of this #3787.
> 
> Update: I have restarted the Linux build.

@kittaakos The build failed.

**Comment by kittaakos** [comment]
Having the macOS and the Windows green builds, I think it is safe to merge this PR. But I leave this decision for the reviewer :)

File: dev-packages/application-manager/package.json (hunk 1 above)
Comment (svenefftinge) [reply dev-packages/application-manager/package.json]: Not now. We decided to not hold back such changes until the foundation has decided how to go forward.

**Pull request merged**